### ISSUE TITLE
Modify temporary shell session options

### ISSUE THREAD

**Issue by QsoulQ** [kingToolbox/WindTerm#2559]
Issue 1: A new shell session is created to connect to the server, and an error message is displayed as shown in the figure. The shell session uses cmd and supports the MAC algorithm on the server, but the connection still fails, but the MAC algorithm is normal when you manually execute the MAC algorithm. Of course, the same error occurs when cmd is used alone, but the ssh session connection is ok. This is not a windterm issue.

Issue 2: If you choose cmd, git, or wls to create a shell session, you cannot use the rzsz command to transfer files. Even if the rz software package has been installed on the server, you can use ssh to log in to the server only by creating a shell session. I know because the above terminal does not support Z modem.

In summary, I would like to say whether the temporary shell session can be modified to use a similar ssh session to connect to the server, which can avoid the above two issue
<img width="579" alt="Snipaste_2024-10-21_15-30-46" src="https://github.com/user-attachments/assets/f359f519-1e29-46ca-9ef3-63eabc899c30">
